src/utils/Sort.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * This code was borrowed from https://material-ui.com/components/tables/
+ */
+
+export type Order = 'asc' | 'desc';
+
+const descendingComparator = <T, >(a: T, b: T, orderBy: keyof T) => {
+  if (b[orderBy] < a[orderBy]) {
+    return -1;
+  }
+  if (b[orderBy] > a[orderBy]) {
+    return 1;
+  }
+  return 0;
+};
+
+export const getComparator = <Key extends keyof any>(
+  order: Order,
+  orderBy: Key,
+): (a: { [key in Key]: any}, b: { [key in Key]: any }) => number => {
+  return order === 'desc' ?
+    (a, b) => descendingComparator(a, b, orderBy) :
+    (a, b) => -descendingComparator(a, b, orderBy);
+};
+
+export const stableSort = <T, >(array: T[], comparator: (a: T, b: T) => number) => {
```

Comment:
provide default comparators for future easier call i.e. `stableSort([1, 2, 3]) should work`

The comparator will just use the data value (1, 2, and 3 in the example above) for comparison with some default ordering (either ASC or DESC)